charts/prometheus-adapter/values.yaml

```diff
@@ -45,6 +42,18 @@ replicas: 1
 podSecurityContext:
   fsGroup: 10001
 
+# SecurityContext of the container
+# ref. https://kubernetes.io/docs/tasks/configure-pod-container/security-context
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop: ["all"]
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 10001
+  # seccompProfile:
+  #   type: RuntimeDefault
```

Comment:
I think we can consider including this as the default since it does provide a security improvement and we're already doing a major version bump.

Reply:
I second that and will update it. Thank you.